plugins/nf-schema/src/resources/META-INF/extensions.idx

```diff
@@ -1 +1,2 @@
-nextflow.validation.SchemaValidator
\ No newline at end of file
+nextflow.validation.ValidationExtension
+nextflow.validation.ValidationObserverFactory
```

Comment:
just to doubleckeck, `ValidationObserver` doesn't have to be added?

Reply:
Nope the factory creates the observer so it gets implemented that way :)